docs/use-cases/README.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases
+  priority: 999
+description: Step-by-step instruction about various use cases
+robots: index, follow
+keywords: help, documentation, use-cases
+---
+# OpenProject Use Cases
+
+
+
+| Use Case                                      | Detailed Description                                         |
+| --------------------------------------------- | ------------------------------------------------------------ |
+| [Portfolio Management](/portfolio-management) | This guide provides detailed step-by-step instruction on how to set up an overview of your project portfolio and create custom reports using the Project Overview, Wiki and the Rich text (WYSIWYG) editor in OpenProject. |
```

Comment:
Unfortunately `[Portfolio Management](/portfolio-management)` on the website would be expanded to `www.openproject.org/portfolio-management` not `www.openproject.org/docs/use-cases/portfolio-management`
You can use the relative dot notation `[Portfolio Management](./portfolio-management)` 
or if it's a children page as in this case you can omit the path `[Portfolio Management](portfolio-management)`